Created Controllers

JdcStig opened the pull request:
Created controllers for each of the models
Added comments to routes explaining the scenario when that route would be called, its expected input and output

JdcStig committed:
Created Controllers

Created controllers for each of the models and added comments to routes explaining the scenario when that route would be called , its expected input and output

seanMooney1996 reviewed (commented):
Hey Jamie, really good work here. The comments are super useful.

Is there anyway you could post some screenshots for evidence of testing? 

And also include some details on how we can test this ourselves?

Roman-Manzhelii reviewed (dismissed):
Good wok Jamie,
I will consider these files when I will create routes

JdcStig committed:
Updated routes to not include duplicated code

added tests to test the following

- Sign up
- Log in
- Create a post
- Create a comment
- Create an image
- Delete a post
- Delete a comment

in root run command npm test and it will run the tests

JdcStig commented:
<img width="792" height="459" alt="image" src="https://github.com/user-attachments/assets/bffc0ca0-0824-4913-82e9-c3f4f9eeb85a" />

JdcStig committed:
Merge branch 'dev' into release1/database/62798/Controllers

JdcStig committed:
commiting package.json as it was being git ignored

JdcStig committed:
added package-lock.json file to .gitignore
